salt/grains/core.py

```diff
@@ -40,11 +41,20 @@
 import salt.utils.pkg.rpm
 import salt.utils.platform
 import salt.utils.stringutils
-from distro import linux_distribution
 from salt.ext import six
 from salt.ext.six.moves import range
 from salt.utils.network import _get_interfaces
 
+
+# rewrite distro.linux_distribution to allow best=True kwarg in version(), needed to get the minor version numbers in CentOS
+def linux_distribution():
```

Comment:
Please prefix this function with `_` since it should not be a _public_ function.

Reply:
done